go.mod

```diff
@@ -132,4 +132,7 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace github.com/rancher/wrangler => github.com/rancher/wrangler v1.1.1-0.20230818201331-3604a6be798d
+replace (
+	github.com/rancher/wrangler => github.com/rancher/wrangler v1.1.1-0.20230818201331-3604a6be798d
+	golang.org/x/net => golang.org/x/net v0.23.0
```

Comment:
just curious, if we would like to bump `golang.org/x/net` to v0.23.0, why don't we just modify the version on the require field?

Reply:
You are right, let me fix that.